src/Lib/Util/Timestamp.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Lib\Util;
+
+use Exception;
+
+class Timestamp
+{
+    /**
+     * Retruns array from date string
+     *
+     * @param  string $str
+     * @return array<int>
+     */
+    public static function timestampToArray(string $str): array
```

Comment:
Did you check https://www.php.net/manual/en/book.datetime.php? If it has all the features, we can use that class instead.